geomas/core/rag_modules/database/chroma_db.py

```diff
@@ -25,7 +25,7 @@ def __init__(
         self,
         client: ChromaDatabaseClient | None = None,
         *,
-        collection_name: str,
+        collection_name: str = "abc",
```

Comment:
make this field mandatory to specify to prevent confusion over collection names
```suggestion
        collection_name: str,
```